app/public/templates/state.html

```diff
@@ -0,0 +1,16 @@
+<script>
+  AFRAME.registerState({
+    initialState: {
+      solvedObjects: {},
+    },
+
+    handlers: {
+      solvedObject: function (state, action) {
+        state.solvedObjects[action.id] = true;
+      },
+      initializeObject: function (state, action) {
+        state.solvedObjects[action.id] = action.solved;
+      },
```

Comment:
Is there a reason this isn't just false? Can an object be initialized in a solved state?

Also, can we not also do action.solved for solvedObject, because I'd think the solved attribute should be true.

If the above is true, then these are identical functions and perhaps can be merged into a single function.

Reply:
For some objects, `initializeObject` is true (for example text-panes). Other objects may also be in a solved state later on.

The reason for the different functions is because I want the code to be easily readable. Like `initializeObject` -> initializes the object. `solveObject` -> solves the object. For now, it would seem we can merge them, but keep in mind later on we will have a different flow for an object being solved vs initialized! (Aka the success flow ticket)